### ISSUE TITLE
Program doesn't open at all

### ISSUE THREAD

**Issue by ToppestOfDogs** [KillzXGaming/Switch-Toolbox#20]
I don't know if I'm doing something wrong, because I can't find any instructions or requirements, but opening the program does nothing and neither does dragging a file on it.

Windows 10 Pro 64Bit Ryzen 2600 GTX 1060 8GB DDR4

**Comment by KillzXGaming** [comment]
Can you try deleting the config.xml file and then boot?

**Comment by ToppestOfDogs** [comment]
That generates a new config but still doesn't open.

**Comment by KillzXGaming** [comment]
Make sure you have 4.6 or higher with installing [this](https://www.microsoft.com/en-us/download/details.aspx?id=53344). If that doesn't boot then try deleting

**Comment by KillzXGaming** [comment]
If it doesn't work. Try [this](https://drive.google.com/file/d/18ogWC6FK_MfPrPSaOwQe3xrguHraz3i4/view?usp=sharing) which will print any errors you get.

**Comment by ToppestOfDogs** [comment]
I already had 4.6, the new version gives me the error: "Could not load file or assembly 'FirstPlugin.Plg, Version=1.0.0.0, Culture=netural, PublicKeyToken=null' or one of its dependencies. Operation is not supported. (Exception from HRESULT: 0x80131515)"

**Comment by ToppestOfDogs** [comment]
I already had 4.6, the new version gives me the error: "Could not load file or assembly 'FirstPlugin.Plg, Version=1.0.0.0, Culture=netural, PublicKeyToken=null' or one of its dependencies. Operation is not supported. (Exception from HRESULT: 0x80131515)"

**Comment by KillzXGaming** [comment]
Make sure FirstPlugin.Plg.dll is in Lib\Plugins. If it's not, make sure to unblock the zip and extract again.

**Comment by ToppestOfDogs** [comment]
Yeah it's in there. This couldn't have anything to do with the program being on E:\ instead of C:\ could it?

**Comment by ToppestOfDogs** [comment]
Nothing seems to be blocking it. The only antivirus I have is Windows Defender and that tells you any time it does anything. I checked and it hasn't blocked anything so I don't think that's the problem.

**Comment by KillzXGaming** [comment]
Does the the program open on the latest? I've made quite a few more improvements, but if it doesn't, try this one? https://drive.google.com/file/d/1F3rDkepxxyJTwBTDPbzCbDsHzyQ83KKq/view?usp=sharing

**Comment by ToppestOfDogs** [comment]
Yes this one does work, thank you.

**Comment by Birdwards** [comment]
I have the same exact problem (with the same exact error message as ToppestOfDogs when I run the error-printing version), but the version in the Google Drive link does not work for me either. I have .NET 4.7 installed, I tried unblocking the ZIP, and FirstPlugin.Plg.dll is in Lib/Plugins. Digging through Event Viewer, I found the following error message from when I attempted to open the latest release build:

Application: Switch_Toolbox.exe
Framework Version: v4.0.30319
Description: The process was terminated due to an unhandled exception.
Exception Info: System.NotSupportedException

Exception Info: System.IO.FileLoadException
   at System.Reflection.RuntimeAssembly._nLoad(System.Reflection.AssemblyName, System.String, System.Security.Policy.Evidence, System.Reflection.RuntimeAssembly, System.Threading.StackCrawlMark ByRef, IntPtr, Boolean, Boolean, Boolean)
   at System.Reflection.RuntimeAssembly.nLoad(System.Reflection.AssemblyName, System.String, System.Security.Policy.Evidence, System.Reflection.RuntimeAssembly, System.Threading.StackCrawlMark ByRef, IntPtr, Boolean, Boolean, Boolean)
   at System.Reflection.RuntimeAssembly.InternalLoadAssemblyName(System.Reflection.AssemblyName, System.Security.Policy.Evidence, System.Reflection.RuntimeAssembly, System.Threading.StackCrawlMark ByRef, IntPtr, Boolean, Boolean, Boolean)
   at System.Reflection.Assembly.Load(System.Reflection.AssemblyName)
   at Switch_Toolbox.Library.PluginLoader.LoadPlugins()
   at Switch_Toolbox.Library.GenericPluginLoader.LoadPlugin()
   at Switch_Toolbox.MainForm..ctor()
   at Switch_Toolbox.Program.Main()

(I know the "Framework Version: v4.0.30319" makes it look like I have an old version of .NET installed, but I looked into that, and I don't think that's the problem. For some reason the files for v4.7 are stored in a folder marked "v4.0.30319". .NET versioning is weird.)

My system:
Windows 10 Home 64-bit, version 1803
Intel Core i5-4460
GeForce GTX 970
8 GB DDR3-1600 RAM
